test/client.ol

```diff
@@ -31,4 +33,8 @@ main {
         .message.pippo.pluto = args[ 0 ]
     }
     writeOnExchange@RabbitMQ( message )
-}
\ No newline at end of file
+}
+
+
+}
+
```

Comment:
the following four jars, are they downloaded with maven? if yes, you can remove it from the commit

Reply:
@klag shell I remove all the jar then?